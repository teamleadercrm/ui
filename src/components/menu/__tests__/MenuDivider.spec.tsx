import React from 'react';
import { render } from '@testing-library/react';
import { MenuDivider } from '..';

describe('Component - MenuDivider', () => {
  it('renders', async () => {
    const { asFragment } = render(<MenuDivider />);
    expect(asFragment()).toMatchSnapshot();
  });
});
