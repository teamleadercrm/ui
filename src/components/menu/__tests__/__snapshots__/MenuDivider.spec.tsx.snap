// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - MenuDivider renders 1`] = `
<DocumentFragment>
  <hr
    class="divider"
    data-teamleader-ui="menu-divider"
  />
</DocumentFragment>
`;
